pkg/skaffold/build/scheduler.go

```diff
@@ -87,6 +105,23 @@ func InParallel(ctx context.Context, out io.Writer, tags tag.ImageTags, artifact
 	return collectResults(out, artifacts, results, outputs)
 }
 
+func artifactWithDepsFor(artifacts []*latest.Artifact) []artifactWithDeps {
```

Comment:
This name isn't great — the second time I read this, I thought this might skip artifacts without dependencies.  `makeChannelModel`?

Reply:
changed struct to `artifactChanModel` and function to `makeArtifactChanModel`. Thanks!